src/components/volunteer/VolunteerEventRecord.tsx

```diff
@@ -152,18 +156,16 @@ export const VolunteerEventRecord = ({
           </View>
         )}
 
-        <Wrapper>
-          <InfoCard category={category} webUrls={webUrls} openWebScreen={openWebScreen} />
-        </Wrapper>
-
-        <Wrapper>
-          {!isAttendingEvent && <RegularText small>{texts.volunteer.attendInfo}</RegularText>}
-          <Button
-            title={isAttendingEvent ? texts.volunteer.notAttend : texts.volunteer.attend}
-            invert={isAttendingEvent}
-            onPress={attend}
-          />
-        </Wrapper>
+        {isAttendingEvent !== undefined && (
```

Comment:
why did you choose `isAttendingEvent !== undefined` here? can we get the same result with `!!isAttendingEvent`?

Reply:
we need to check if `isAttendingEvent` is not set, which means its neither `true` or `false`. if it is `true` or `false` their was a successful validation of the attending status, otherwise we do not have a logged in user, so we will not show any buttons.

this is what i meant with `- added condition to not show event attending button if not logged in` in the corresponding commit.